src/main/java/robot/MainTest.java

```diff
@@ -0,0 +1,27 @@
+package robot;
```

Comment:
Why isn't this file inside the `pure_pursuit` package?
```suggestion
package robot.subsystems.drivetrain.pure_pursuit;
```